phpunit.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="vendor/autoload.php"
```

Comment:
Please create a file under

* `tests/bootstrap.php`

This will load the `vendor/autoload.php` and also the `credentials.php` file along with the library itself.

This way we wouldn't have to include the library again when writing new tests.